maggy/core/executors/tf_dist_executor.py

```diff
@@ -78,6 +78,8 @@ def wrapper_function(_: Any) -> None:
         reporter = Reporter(log_file, partition_id, 0, __builtin__.print)
         builtin_print = __builtin__.print
 
+        trial_id, parameters = client.get_suggestion(reporter)
```

Comment:
Why do you need to get a suggestion for distribtued training? thought that's only for optimization and ablation?